assets/scss/_variables_project.scss

```diff
@@ -49,3 +52,45 @@ h2, h3 {
 {
   clear: both;
 }
+
+#td-section-nav {
```

Comment:
ah - nice! i tried to this extra level of nesting to work as a "suggestion" in GH, but didn't know SCSS foo well enough.  this looks much better! I'll suck down the branch and test locally real quick.